File: docs/user/en/query/filtering/examples/logical/implicit-binding.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					entityPrimaryKeyInSet(110066, 106742),
					attributeEquals("code", "lenovo-thinkpad-t495-2")
				)
			)
		);
	}
);

File: documentation/user/en/query/filtering/examples/logical/implicit-binding.cs
EvitaResponse<SealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		EntityPrimaryKeyInSet(110066, 106742),
        		AttributeEquals("code", "lenovo-thinkpad-t495-2")
        	)
        )
	)
);